browser/extensions/api/settings_private/brave_prefs_util.cc

```diff
@@ -66,6 +66,8 @@ const PrefsUtil::TypedPrefMap& BravePrefsUtil::GetWhitelistedKeys() {
       settings_api::PrefType::PREF_TYPE_NUMBER;
   (*s_brave_whitelist)[kAskWidevineInstall] =
       settings_api::PrefType::PREF_TYPE_BOOLEAN;
+  (*s_brave_whitelist)[kAutocompleteDisabled] =
+      settings_api::PrefType::PREF_TYPE_BOOLEAN;
```

Comment:
What does adding the preference here do?

Reply:
With this, settings option can work together with this pref.
So, we don't need to do set/get explicitly for it. Ex, toggle button do it instead of us.
AFAIK, this only works for profile prefs. So, I did manually for local state pref in tor PR.

Reply:
ah - ok I remember @petemill mentioning this 👍

Reply:
ah - ok I remember @petemill mentioning this 👍